src/style.css

```diff
@@ -0,0 +1,145 @@
+html {
+  font-family: Roboto, Helvetica, sans-serif;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  
+  width: 200px;
+  height: 406px;
+}
+
+.card_logo {
+  width: 160px;
+  height: 134px;
+  padding: 33px 19px 0;
+}
+
+.card_name {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+
+  padding-top: 40px;
+  margin: 0;
+
+  width: 166px;
+  height: 36px;
+  
+  color: #060b35;
+}
+
+.card_code {
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+
+  padding: 4px 84px 0 16px;
+  margin: 0;
+
+  width: 98px;
+  height: 14px;
+
+  color: #616070;
+}
+
+.stars {
+  display: flex;
+  justify-content: center;
+  background-position: center;
+  
+  padding-top: 18px;
+  padding-left: 2px;
+}
+
+.stars__star {
+  background-image: url(images/star.svg);
+  background-repeat: no-repeat;
+  
+  width: 16px ;
+  height: 16px;
+}
+
+.stars_reviews {
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+
+  padding-left: 20.7px;
+
+  margin: 0;
+
+  color: #060b35;
+}
+
+.stars__star:first-child,
+.stars__star:nth-child(-n+2),
+.stars__star:nth-child(-n+3),
+.stars__star:nth-child(-n+4) {
+  background-image: url(./images/star-active.svg);
+  background-repeat: no-repeat;
+  
+  width: 16px ;
+  height: 16px;
+ 
+  margin-right: 4px;
```

Comment:
Remove repeatable code

Reply:
Hello, all tests passed. Links doesn't work because of problems in GitHub.